source/tutorials/importing.rst

```diff
@@ -84,12 +85,14 @@ Format description
 
 Paired-end "`Earth Microbiome Project (EMP) protocol`_" formatted reads should have three ``fastq.gz`` files total:
 
-1. one ``fastq.gz`` file that contains the forward sequence reads,
-2. one ``fastq.gz`` file that contains the reverse sequence reads,
+1. one ``forward.fastq.gz`` file that contains the forward sequence reads,
+2. one ``reverse.fastq.gz`` file that contains the reverse sequence reads,
 3. and a third that contains the associated barcode reads
 
 In this format, sequence data is still multiplexed (i.e. you have only one forward and one reverse ``fastq.gz`` file containing raw data for all of your samples).
 
+Because you are importing multiple files in a directory, the filenames `forward.fastq.gz` and `reverse.fastq.gz` are *required*.
```

Comment:
```suggestion
Because you are importing multiple files in a directory, the filenames ``forward.fastq.gz``, ``reverse.fastq.gz``, and ``barcodes.fastq.gz`` are *required*.
```